google/cloud/spanner/commit_options.h

```diff
@@ -94,12 +95,25 @@ class CommitOptions {
     return transaction_tag_;
   }
 
+  // Set the max commit delay of the `spanner::Client::Commit()` call.
+  CommitOptions& set_max_commit_delay(
+      absl::optional<std::chrono::milliseconds> max_commit_delay) {
+    max_commit_delay_ = std::move(max_commit_delay);
+    return *this;
+  }
+
+  // The max commit delay for the `spanner::Client::Commit()` call.
+  absl::optional<std::chrono::milliseconds> max_commit_delay() const {
```

Comment:
On second thought, let's change this to return a `absl::optional<std::chrono::milliseconds> const&`, to match `transaction_tag()`.

Reply:
Done.